docs/CRYPTEXT_DICTIONARY.md

```
@@ -0,0 +1,672 @@
+| `uint8(offset)` | 0 | `functions.insert("uint8".to_string(), 0)` | ✅ |
+| `uint16(offset)` | 1 | `functions.insert("uint16".to_string(), 1)` | ✅ |
+| `uint32(offset)` | 2 | `functions.insert("uint32".to_string(), 2)` | ✅ |
+| `uint16be(offset)` | 3 | `functions.insert("uint16be".to_string(), 3)` | ✅ |
+| `uint32be(offset)` | 4 | `functions.insert("uint32be".to_string(), 4)` | ✅ |
+| `int8(offset)` | 5 | `functions.insert("int8".to_string(), 5)` | ✅ |
+| `int16(offset)` | 6 | `functions.insert("int16".to_string(), 6)` | ✅ |
+| `int32(offset)` | 7 | `functions.insert("int32".to_string(), 7)` | ✅ |
+| `int16be(offset)` | 8 | `functions.insert("int16be".to_string(), 8)` | ✅ |
+| `int32be(offset)` | 9 | `functions.insert("int32be".to_string(), 9)` | ✅ |
+
+**Built-in Function Coverage: 100%**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)